Jorge/HTML & CSS/index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>First Task</title>
+</head>
+<body>
+  <div class="main-container">
+    <nav class="header">
+      <div class="header__logo">
+        <img src="images/broly.png" alt="logo">
+      </div>
+      <div class="header__menu">
+        <ul>
+          <li><a href="#">FEATURES</a></li>
+          <li><a href="#">TEMPLATES</a></li>
+          <li><a href="#">INTEGRATION</a></li>
+          <li><a href="#">PRICING</a></li>
+        </ul>
+      </div>
+      <div class="header__button">
+        <button class="button">SIGN IN</button>
+        <button class="button primary">SIGN UP</button>
+      </div>
+    </nav>
+    <div class="hero">
+      <h3>Hola, me llamo</h3>
+      <h1>Jorge</h1>
+    </div>
+    <section class="achivements">
+      <div class="achivements__item">
+        <span>10</span>
+        Años de experiencia
+      </div>
+      <div class="achivements__item">
+        <span>10</span>
+        Galardones
+      </div>
+      <div class="achivements__item">
+        <span>10</span>
+        Clientes satisfechos
+      </div>
+      <div class="achivements__item">
+        <span>10</span>
+        Proyectos concluidos
+      </div>
+    </section>
+    <section class="technologies">
+      <article class="technologies__blog">
+        <h2>THE AMAZING <br>HUMBLE</h2>
+        <h2>Spend less Time Coding</h2>
```

Comment:
Same as with the text inside the Hero component. Lets try to achieve the same look with only a single h1 with a span inside, both with their own classes to style the font-size, width and line-height